pladder/bot.py

```diff
@@ -45,6 +45,24 @@ def new_command_group(self, name):
         return self.commands.new_command_group(name)
 
     def RunCommand(self, timestamp, network, channel, nick, text):
+        prefilter_result = self.RunCommand_internal(timestamp, network, channel, nick, text)
+        if prefilter_result['command'] != "error" and self.commands.lookup_command("output-filter"):
+            try:
+                filter_result = self.RunCommand_internal(timestamp, network, channel, nick, "output-filter {{{}}}".format(prefilter_result['text']))
```

Comment:
This way of composing a command by concatenating strings has issues. It's possible to "escape the argument" by having something like `"} blah {"` in the string. Compare to SQL injections.

The proper way would be to use the `CommandBinding` object you get out from the `lookup_command()` call above and use `pladder.script.interpreter.apply_call()` to run the command with the string as its argument:

    filter_command = self.commands.lookup_command("output-filter")
    if filer_command is not None:
        post_filter_result_text, post_filter_display_name = apply_call(context, command, "output-filter", [result_text])